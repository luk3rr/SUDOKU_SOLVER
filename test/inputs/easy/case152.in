005000000 000850041 180007250 860700000 030001000 000028010 900002060 000090530 002000970 
