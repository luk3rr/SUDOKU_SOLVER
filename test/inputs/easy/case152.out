Solving the following grid:
0 0 5 | 0 0 0 | 0 0 0 
0 0 0 | 8 5 0 | 0 4 1 
1 8 0 | 0 0 7 | 2 5 0 
------+-------+------
8 6 0 | 7 0 0 | 0 0 0 
0 3 0 | 0 0 1 | 0 0 0 
0 0 0 | 0 2 8 | 0 1 0 
------+-------+------
9 0 0 | 0 0 2 | 0 6 0 
0 0 0 | 0 9 0 | 5 3 0 
0 0 2 | 0 0 0 | 9 7 0 


Solution found :')

4 7 5 | 2 1 6 | 8 9 3 
3 2 6 | 8 5 9 | 7 4 1 
1 8 9 | 4 3 7 | 2 5 6 
------+-------+------
8 6 1 | 7 4 5 | 3 2 9 
2 3 7 | 9 6 1 | 4 8 5 
5 9 4 | 3 2 8 | 6 1 7 
------+-------+------
9 4 3 | 5 7 2 | 1 6 8 
7 1 8 | 6 9 4 | 5 3 2 
6 5 2 | 1 8 3 | 9 7 4 

Algorithm: BFS
Total time: 109 ms
Total expanded states: 16804
